### DISCUSSION TITLE
Multiple modifier keys?

### DISCUSSION THREAD

**Discussion by neonWhiteout** [sezanzeb/input-remapper#39] [Q&A] [answered]
I use control + alt + t as a shortcut for opening a terminal, wondering if there's a way I can map this onto a button on my mouse?  From what I understand, you can only use one modifier.

**Comment by sezanzeb** [comment, marked as answer]
Either with `m(control_l, m(alt_l, k(t)))` or in the upcoming version with `control_l + alt_l + t`. If you are installing from the main branch you can already do that.

**Reply by neonWhiteout** [reply]
Ah, thank you.
